internal/nodes/crd_translator.go

```diff
@@ -31,6 +31,37 @@ import (
 	"github.com/avinetworks/sdk/go/models"
 )
 
+func removeVSNodeRefs(crd string, vsNode *AviVsNode) {
+	// Fields based off Hostrule CRD
+	if crd == lib.HostRule {
+		vsNode.Enabled = nil
+		vsNode.SSLKeyCertAviRef = ""
+		vsNode.WafPolicyRef = ""
+		vsNode.HttpPolicySetRefs = []string{}
+		vsNode.AppProfileRef = ""
+		vsNode.AnalyticsProfileRef = ""
+		vsNode.ErrorPageProfileRef = ""
+		vsNode.SSLProfileRef = ""
+		vsNode.VsDatascriptRefs = []string{}
+		if vsNode.ServiceMetadata.CRDStatus.Value != "" {
+			vsNode.ServiceMetadata.CRDStatus.Status = "INACTIVE"
+		}
+	}
+
+	// fields based off HTTPRule CRD
+	if crd == lib.HTTPRule {
```

Comment:
We should either return from the previous if, or make this an `else if` condition.